### PR TITLE
NO-JIRA:docu change: fix find command

### PR DESCRIPTION
**Pull request by Unfaced85**
After testing the Migration part out I found out that the find command does not find the secrets, instead it finds the CredentialsRequest Objects.

This change in the Documentation will reflect how it should work.

### CODE REVIEW

File: docs/sts.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ $ ./ccoctl aws create-iam-roles --output-dir ./newstscluster/ --name newstsclust
 
 13. Apply the Secrets generated by the above command:
 ```bash
-$ find ./cred-reqs -iname "*yaml" -print0 | xargs -I {} -0 -t oc replace -f {}
+$ find ./newstscluster -iname "*yaml" -print0 | xargs -I {} -0 -t oc replace -f {}
```
Comment (jstuever) [inline docs/sts.md:179]: This can be more precise by using `find ./newstscluster/manifests`.

**Review by jstuever — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request. One minor adjustment and then we should be good to go.

File: docs/sts.md (hunk 1 above)
Comment (Unfaced85) [reply docs/sts.md]: updated as requested

**Review by jstuever — APPROVED** [review]
/lgtm
/approve

**Pull request merged**